lib/active_admin_csv_import/dsl.rb

```diff
@@ -87,6 +88,12 @@ def update_row_resource(resource, params)
           resource.save
         end
 
+        def build_row_resource
+          resource = end_of_association_chain.new
+          # controller before create callback
+          resource = before_create(resource) if respond_to? :before_create
```

Comment:
This feels a bit odd. Should at least be namespaced to CSV. Was this something here already?